bindings/nodejs/lib/types/wallet/transaction.ts

```diff
@@ -13,22 +18,24 @@ export enum InclusionState {
 }
 
 /** A Transaction with metadata */
-export interface Transaction {
+export class Transaction {
     /** The transaction payload */
-    payload: ITransactionPayload;
+    @Type(() => TransactionPayload)
+    payload!: TransactionPayload;
```

Comment:
What's the ! for?

Reply:
! means mandatory

Reply:
I'm confused, if ? means optional and ! means mandatory, what does nothing mean?

Reply:
oh apparently that means "cannot be null", hmmm thats no good

Reply:
hm to remove the ! i would have to add constructors everywhere, thats a bit annoying as we dont actually use those (as theyre deserialized). what do you want me to do?

Reply:
weve decided its fine